CMakeLists.txt

```diff
@@ -10,6 +10,10 @@ set(CMAKE_CXX_STANDARD 14)
 set(CMAKE_CXX_STANDARD_REQUIRED ON)
 set(CMAKE_CXX_EXTENSIONS OFF)
 
+#pkg-config values
+set(DXHEADERS_URL "https://github.com/microsoft/DirectX-Headers")
+set(DXHEADERS_DESCRIPTION "Headers for using D3D12")

```

Comment:
Why not just inline this in the .pc.in template, if that's the only place they are used?